libs/ledgerjs/packages/react-native-hw-transport-ble/src/BleTransport.ts

```diff
@@ -358,6 +377,10 @@ async function open(deviceOrId: Device | string, needsReconnect: boolean) {
     return open(device, false);
   }
 
+  // Nb If this priority request is executed on an unpaired device and we refuse the pairing
+  // it throws an internal crash from BleManager. In order to avoid that, only request the
+  // priority after we know we are paired. Leveraging the inferMTU success here.
+  await transport.requestConnectionPriority("High");
```

Comment:
IIRC it was this connection priority that was creating the freeze on the Firmware update over BLE feature. We'll be reintroducing the bug if we add it back. Why is it needed?

Reply:
It's not needed, this is an old PR and it's good you reviewed it.

Reply:
in fact it just needed a rebase